libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/NBGeocoderMetadataParser.m

```diff
@@ -0,0 +1,74 @@
+//
+//  NBGeocoderMetadataParser.m
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBGeocoderMetadataParser.h"
+
+@implementation NBGeocoderMetadataParser {
+@private
+  SQLiteDatabaseConnection *_databaseConnection;
+}
+
+- (instancetype)init {
+  self = [super init];
+  return self;
+}
+
+- (void)convertFileToSQLiteDatabase:(NSString *)completeTextFilePath
```

Comment:
I'm confused. What's this parameter for? what's the difference between this one and the file name one?

Reply:
One has the whole filepath: (Ex: rastaarhaghi/documents/geocoding/en/1.txt. the other just has the 1.txt